ros1_foxglove_bridge/src/ros1_foxglove_bridge_nodelet.cpp

```diff
@@ -805,8 +809,11 @@ class FoxgloveBridge : public nodelet::Nodelet {
       const auto param = fromRosParam(paramName, paramValue);
       _server->updateParameterValues({param});
     } catch (const std::exception& ex) {
-      ROS_ERROR("Failed to convert parameter: %s", ex.what());
-      return;
+      ROS_ERROR("Failed to update parameter: %s", ex.what());
+    } catch (const XmlRpc::XmlRpcException& ex) {
+      ROS_ERROR("Failed to update parameter: %s", ex.getMessage().c_str());
+    } catch (...) {
+      ROS_ERROR("Failed to update parameter");
```

Comment:
Is there any way to include more info about the exception type in the log? I googled a little bit and couldn't find anything... I see that you can use `current_exception()` to get an `exception_ptr`, but then there's nothing useful you can do with that 😬

Reply:
I'm also not aware that this would be possible. Also just learned that you can literally throw anything, e.g. just `throw 1;`